.rustfmt.toml

```diff
@@ -1,6 +1,2 @@
-# Must be run in nightly:
-# cargo +nightly fmt
-
 reorder_imports = true
-max_width = 100
-imports_granularity = "One"
\ No newline at end of file
+max_width = 100
```

Comment:
Undo?

Reply:
I removed  `imports_granularity = "One"` which requires nightly and is apparently only used by me haha

Reply:
Yeah sorry for not using it, but I still think it has some value, similar to clippy warnings which we fix once in a while. And if you want to remove it it's better to open a separate PR